lib/js/icons/fontawesome.ts

```diff
@@ -347,7 +348,7 @@ export const FONTAWESOME_ICONS = {
 	FA_SHARE: faShare,
 	FA_SHARE_NODES: faShareNodes,
 	FA_SHIELD_HALVED: faShieldHalved,
-	FA_SIDEBAR: faSidebar,
+	FA_SIDEBAR_FLIP: faSidebarFlip,
```

Comment:
Do we need to import flipped icons? Are we going to import flipped and non flipped version of this asset when we find a use-case for for the latter version? Why we don't use `FA_SIDEBAR` with `flippedHorizontal` prop?

Reply:
`IconButton` doesn't have `flippedHorizontal` prop.

Reply:
maybe it's time to expose it

Reply:
I'm cool with that but you should probably talk with UX. 🙂

Reply:
https://bethinkteam.slack.com/archives/CFBLF5RGA/p1721897628795609